skt/reporter.py

```diff
@@ -141,6 +142,11 @@ def __init__(self, cfg):
         # We need to save the job IDs when iterating over state files when
         # multireporting
         self.multi_job_ids = []
+        # We need to know if we're looking for soaking tests in xml using redis
+        # or not
+        self.soak = not cfg.get('nosoak')
```

Comment:
Is there a way we could adjust this config option to avoid a double negative? It seems weird to say `not nosoak`.